app/data/JsonConversions.scala

```diff
@@ -11,6 +11,10 @@ object JsonConversions {
 
   implicit val category = Writes[Category](category => JsString(category.name.toLowerCase))
 
+  implicit val privacyStatusWrites: Writes[PrivacyStatus] = Writes[PrivacyStatus](privacyStatus => JsString(privacyStatus.name.toLowerCase))
+
+  implicit val privacyStatusReads: Reads[PrivacyStatus] = __.read[PrivacyStatus]
```

Comment:
For thrift enums I've been creating scala enums like this: https://github.com/guardian/media-atom-maker/blob/master/app/model/Category.scala